README.md

```diff
@@ -0,0 +1,45 @@
+# MY INF 7420 PORTFOLIO PROJECT
+I am going to build a portfolio website for this project. It will showcase my work and help me get a job after graduation.
+
+## Wireframes
+I value planning so here are some wireframes to show the site architecture and layout could possibly be.
+
+![Wireframe of the Home Page](wireframes/Home.pdf)
```

Comment:
Please update the image references to point to the jpg and not pdf versions of the wireframes.

Reply:
Ok, I think I fixed it.